tests/test_schema_validator.py

```diff
@@ -56,7 +56,7 @@ def test_schema():
     with open("schemas/questionnaire_v1.json", encoding="utf8") as schema_data:
         schema = json.load(schema_data)
         resolver = RefResolver(
-            base_uri="https://eq.ons.gov.uk/",
+            base_uri="",
```

Comment:
How does json schema resolve the base_uri now that it's been removed here?

Reply:
it uses the relative path now.

Reply:
Oh yeah ofc, thanks 😆